packages/devtools_app/test/integration_tests/server_test.dart

```diff
@@ -32,36 +32,22 @@ final List<int> browserPids = [];
 void main() {
   final bool testInReleaseMode =
```

Comment:
It's used to skip the server API tests when running on the ddc bot (since our server API bits only work in release mode) since the results would be the same (and both would use the release build).